api/CustomElementRegistry.json

```diff
@@ -247,7 +241,18 @@
             "safari_ios": {
               "version_added": "10.1",
               "notes": "Supports 'Autonomous custom elements' but not 'Customized built-in elements'"
```

Comment:
Now that I think about it, Safari should also have `"partial_implementation": true`.